### PR TITLE
[DEV APPROVED] update webchat.js

### PR DESCRIPTION
**Pull request by pmcostello**
update web chat.js file to stop site monitoring timing out

---
This change is [Reviewable](https://reviewable.io/reviews/moneyadviceservice/frontend/1734)

### CODE REVIEW

File: vendor/assets/javascripts/webchat.js [hunk 1]
```diff
@@ -94,7 +94,7 @@ function sWOTrackPage() {
       sWOChatElement = document.getElementById('whoson_chat_link');
     }
     sWOUrl += "&response=g";
-    sWOChatstart += "?domain=" + sWODomain;
+    -1 == sWOChatstart.indexOf("domain") && (sWOChatstart += "?domain=" + sWODomain)
```
Comment (Tr4pSt3R) [inline vendor/assets/javascripts/webchat.js]: Does this need to be passed to the _Frontend_ team?

**Comment by Tr4pSt3R** [comment]
👍

File: vendor/assets/javascripts/webchat.js (hunk 1 above)
Comment (pmcostello) [inline vendor/assets/javascripts/webchat.js]: don't think so, it's already been fully tested on UAT, this edit is recommended by the web chat supplier to stop the site monitoring timing out, although I would prefer one of the FE team to review and sign off on it though

**Comment by tomas-stefano** [comment]
👍 👍 Can you add the motive of the update to the commit message too, please? It will be easier to understand the reason of the change in some future not far away :)

**Comment by pmcostello** [comment]
Sure, np, will do

**Comment by pmcostello** [comment]
commit message updated, squashing and merging now

**Pull request merged**